include/internal/refcount.h

```diff
@@ -134,14 +134,14 @@ static __inline int CRYPTO_DOWN_REF(volatile int *val, int *ret,
 static __inline int CRYPTO_UP_REF(volatile int *val, int *ret,
                                   ossl_unused void *lock)
 {
-    *ret = _InterlockedExchangeAdd(val, 1) + 1;
+    *ret = _InterlockedExchangeAdd((long volatile*) val, 1) + 1;
```

Comment:
nit: shouldn't the cast be written as `(long volatile *)val`? (i.e. with space before `*`, and no space after `)`)